screens/CameraScreen.tsx

```diff
@@ -0,0 +1,181 @@
+import * as React from 'react';
+import { useFocusEffect } from '@react-navigation/native';
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
+import { Camera } from 'expo-camera';
+import { Ionicons, MaterialIcons, MaterialCommunityIcons } from '@expo/vector-icons';
+
+export default function CameraScreen({ navigation: { goBack } }) {
+
+  const [camera, setCamera] = React.useState<Camera | null>(null);
+  const [hasPermission, setHasPermission] = React.useState<any>(null);
+  const [type, setType] = React.useState<any>(Camera.Constants.Type.back);
```

Comment:
probably `useState<Camera.Constants.Type>(Camera.Constants.Type.back)` if `Camera.Constants.Type` is an enum

Reply:
Type is any in the source code

![image](https://user-images.githubusercontent.com/37185367/111246401-8ffdb400-85dc-11eb-8950-58c98c89d97d.png)

Reply:
if the only types that we care about are `back` and `front`, you could make a custom type (or enum)
```
type CameraType = Camera.Constants.Type.back | Camera.Constants.Type.front;
```
and use `useState<CameraType>(Camera.Constants.Type.back)`

this provides a bit more transparency

Reply:
alright sounds good